tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -352,6 +352,15 @@ dash/test_relaxed_match_negative.py:
     conditions:
       - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/16407"
 
+#######################################
+#####       db_migrator           #####
+#######################################
+db_migrator/test_db_migrator.py::test_migrate_dns_03:
```

Comment:
The prefix is wrong. Should be `db_migrator/test_migrate_dns.py::test_migrate_dns_03:`

Reply:
Thanks a lot!